File: 프로그래머스/lv1/81301. 숫자 문자열과 영단어/숫자 문자열과 영단어.py
def solution(s):
    
    answer=s
    en={"zero":0,"one":1,"two":2,"three":3,"four":4,"five":5,"six":6,"seven":7,"eight":8,"nine":9}
    
    for word in en :
        if word in answer :
            answer=answer.replace(word,str(en[word]))
    
    return int(answer)

File: 프로그래머스/1/81301. 숫자 문자열과 영단어/숫자 문자열과 영단어.java
import java.util.*;
class Solution {
    public int solution(String s) {
        int answer = 0;
        
        HashMap<String,Integer> map=new HashMap<>();
        map.put("zero",0);
        map.put("one",1);
        map.put("two",2);
        map.put("three",3);
        map.put("four",4);
        map.put("five",5);
        map.put("six",6);
        map.put("seven",7);
        map.put("eight",8);
        map.put("nine",9);
        
        int p=0;
        int q=p+3;
        
        Set<String> keySet=map.keySet();
        for(String key:keySet){
            s=s.replace(key,map.get(key).toString());
        }
        
        answer=Integer.parseInt(s);
        
        return answer;
    }
}